workers_test.go

```diff
@@ -823,3 +826,37 @@ func TestUploadWorker_WithServiceBinding(t *testing.T) {
 	})
 	assert.NoError(t, err)
 }
+
+func TestUploadWorker_WithLogpush(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/accounts/"+testAccountID+"/workers/scripts/foo", func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodPut, r.Method, "Expected method 'PUT', got %s", r.Method)
+		mpUpload, err := parseMultipartUpload(r)
+		assert.NoError(t, err)
+
+		expected := true
+		assert.Equal(t, &expected, mpUpload.Logpush)
+
+		w.Header().Set("content-type", "application/json")
+		fmt.Fprint(w, uploadWorkerResponseData)
+	})
+	logpush := true
+	res, err := client.UploadWorker(context.Background(), AccountIdentifier(testAccountID), CreateWorkerParams{ScriptName: "foo", Script: workerScript, Logpush: &logpush})
```

Comment:
```suggestion
	res, err := client.UploadWorker(context.Background(), AccountIdentifier(testAccountID), CreateWorkerParams{ScriptName: "foo", Script: workerScript, Logpush: BoolPtr(true)})
```

we have [`BoolPtr`](https://github.com/cloudflare/cloudflare-go/blob/master/convert_types.go#L77) helper to simlify this sort of usage.